core/frontend/web/middleware/handle-image-sizes.js

```diff
@@ -7,6 +7,8 @@ const activeTheme = require('../../services/theme-engine/active');
 const config = require('../../../shared/config');
 
 const SIZE_PATH_REGEX = /^\/size\/([^/]+)\//;
+// When formatting, we need to append the right extension to the file name, because the mime type is interpreted from the file extension, and not the file contents
+const FORMAT_PATH_REGEX = /_format\.([^/.]+)$/;
```

Comment:
I think we can remove the _ here?

Reply:
Oh I might be misunderstanding. I thought this was for a `/format/` path like `/size/` but it looks like its part of the filename...

Reply:
Change was not yet pushed! Solved now :)